src/main.test.js

```diff
@@ -25,7 +25,7 @@ test("Valid input with restricted choices", () => {
 
 test("Valid input with default value for type", () => {
   const testEnv = "DEBUG: [0, 1]=1";
-  expect(envSpecToHTML(testEnv)).toEqual(
+  expect(envSpec.parse(testEnv)).resolves.toEqual(
```

Comment:
You need to `return` this, in order to mark the test successfully as passed/failing ([see example](https://jestjs.io/docs/en/expect.html#resolves)).